### PR TITLE
Allow receiving input from env vars

### PR DESCRIPTION
**Pull request by itasahobby**
related to #6

### CODE REVIEW

File: firestarter/cli/parser.py [hunk 1]
```diff
@@ -27,15 +27,28 @@ def main():
...
+    config_file = None
+
+    input_vars = os.environ.get("INPUT_VARS", None)
+    input_secrets = os.environ.get("INPUT_SECRETS", None)
+    input_config_file = os.environ.get("INPUT_CONFIG_FILE", None)
+
+    if None not in (input_vars, input_secrets, input_config_file):
+      vars.update(tomllib.loads(input_vars))
+      secrets.update(tomllib.loads(input_secrets))
+      config_file = input_config_file if input_config_file is not None else args.config_file
```
Comment (alambike) [inline firestarter/cli/parser.py:41]: I would use an if for every input, if not this if is going to be skipped with any of the inputs being None

File: firestarter/cli/parser.py [hunk 2]
```diff
@@ -27,15 +27,28 @@ def main():
...
+
     if args.vars_inline or args.secrets_inline:
       logger.info(f"Inline args detected")
-      vars = tomllib.loads(args.vars_inline).get("vars") or {}
+      vars.update(tomllib.loads(args.vars_inline).get("vars"))
```
Comment (alambike) [inline firestarter/cli/parser.py:45]: How the updates behaves? Which config has higher precedence, the new or the old?

**Review by alambike — CHANGES_REQUESTED** [review]

**Comment by itasahobby** [comment]
Made the use of arguments optional, since now they are only meant to be used on local environments (using environment variables for ci cd).

Removed legacy `--vars` and `--secrets` allowing only inline table toml format

File: firestarter/cli/parser.py [hunk 3]
```diff
@@ -20,23 +20,25 @@ def main():
     arg_parser = argparse.ArgumentParser()
 
     arg_parser.add_argument('workflow', type=str, help='Name of the workflow to run')
-    arg_parser.add_argument('--vars', type=str, help='Variables to pass to the workflow, in toml format')
-    arg_parser.add_argument('--vars_inline', type=str, help='Variables to pass to the workflow, in inline table toml format')
-    arg_parser.add_argument('--secrets', type=str, help='Secrets to pass to the workflow, in toml format')
-    arg_parser.add_argument('--secrets_inline', type=str, help='Secrets to pass to the workflow, in inline table toml format')
+    arg_parser.add_argument('--vars', type=str, help='Variables to pass to the workflow, in inline table toml format')
```
Comment (alambike) [inline firestarter/cli/parser.py:23]:
```suggestion
    arg_parser.add_argument('--vars', type=str, help='Variables to pass to the workflow, in inline table toml format. Example: --vars="vars = { var1: \"value1\", var2: \"value2\"}"')
```

**Review by alambike — APPROVED** [review]
LGTM

**Pull request merged**